### PR TITLE
Replace gh-create-issue by a shell script

### PR DESCRIPTION
**Pull request by cryptobioz**

### CODE REVIEW

File: gh-create-issues.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+url="$API_BASE_URL/repos/$REPO_OWNER/$REPO_NAME/issues?token=$TOKEN"
+
+issues=$(curl -q "$url" | jq -r ".[].title" )
```
Comment (mcanevet) [inline gh-create-issues.sh]: Need pagination management

File: gh-create-issues.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+while read line; do
+  exists=0
+  title=$(echo "$line" | jq -r ".title")
+
+  while read issue; do
+    if [ "$issue" = "$title" ]; then
+      exists=1
```
Comment (mcanevet) [inline gh-create-issues.sh:26]: Break?

**Review by mcanevet — CHANGES_REQUESTED** [review]

File: gh-create-issues.sh (hunk 2 above)
Comment (cryptobioz) [reply gh-create-issues.sh:26]: Done in 51c20ca.

File: gh-create-issues.sh (hunk 1 above)
Comment (cryptobioz) [reply gh-create-issues.sh]: Done in 51c20ca.

**Comment by mcanevet** [comment]
+1

**Review by mcanevet — APPROVED** [review]

**Pull request merged**